### PR TITLE
Document Lists API

### LINKED ISSUE #176: Document lists API
The current lists API (see `zio.redis.api.Lists`) exposes functions with argument names that are generic and meaningless. I propose to rename those to match those used in the [official documentation](https://redis.io/commands). Additionally, it would be great to provide a small, single-sentence scaladoc for them.

### PR DESCRIPTION
**Pull request by helanto**
closes #176

### CODE REVIEW

File: redis/src/main/scala/zio/redis/api/Lists.scala [hunk 1]
```diff
@@ -10,47 +10,200 @@ import zio.{ Chunk, ZIO }
...
-
-  final def blPop(a: String, as: String*)(b: Duration): ZIO[RedisExecutor, RedisError, Option[(String, String)]] =
-    BlPop.run(((a, as.toList), b))
-
-  final def brPop(a: String, as: String*)(b: Duration): ZIO[RedisExecutor, RedisError, Option[(String, String)]] =
-    BrPop.run(((a, as.toList), b))
-
-  final def lInsert(a: String, b: Position, c: String, d: String): ZIO[RedisExecutor, RedisError, Long] =
-    LInsert.run((a, b, c, d))
+  /** Pops an element from the list stored at source, pushes it to the list stored at destination; or block until one
```
Comment (mijicd) [inline redis/src/main/scala/zio/redis/api/Lists.scala:13]: I would make a new line here, e.g.:

```
/**
 * Pops an element ...
 * ...
 */
```

**Review by mijicd — COMMENTED** [review]
Just a small remark, other than that looks superb!

File: redis/src/main/scala/zio/redis/api/Lists.scala (hunk 1 above)
Comment (helanto) [reply redis/src/main/scala/zio/redis/api/Lists.scala:13]: I am fixing that!

**Review by mijicd — APPROVED** [review]
I'm merging it as soon as build is green. Thanks for taking care of this! :pray:

**Pull request merged**